src/common/dcsvg.cpp

```diff
@@ -108,6 +118,127 @@ wxString wxBrushString(wxColour c, int style = wxBRUSHSTYLE_SOLID)
     return s;
 }
 
+wxString wxGetPenPattern(wxPen& pen)
+{
+    wxString s;
+    float w = pen.GetWidth();
+    if (pen.GetWidth() == 0)
+        w = 1;
+    w = w / 3;
```

Comment:
I wonder where do all these factors (3 here, 2, 5, 8, 10 below) come from? It would be nice to have a comment explaining it...